src/OrchardCore.Modules/OrchardCore.Lucene/Deployment/LuceneIndexRebuildDeploymentStep.cs

```diff
@@ -0,0 +1,19 @@
+using OrchardCore.Deployment;
+
+namespace OrchardCore.Lucene.Deployment
+{
+    /// <summary>
+    /// Adds rebuild lucene index task to a <see cref="DeploymentPlanResult"/>.
```

Comment:
Do something similar everywhere, with "Lucene" indexing:

```suggestion
    /// Adds rebuild Lucene index task to a <see cref="DeploymentPlanResult"/>.
```